android-generator/src/test/scala/models/generator/android/TestAndroidClasses.scala

```diff
@@ -130,12 +129,13 @@ class TestAndroidClasses
     val result = AndroidClasses.invoke(InvocationForm(models.TestHelper.service(json.format())))
 
     result.isRight should be(true)
```

Comment:
I guess an FYI, one could replace
```
result.isRight should be(true)
val files = result.right.get
```
with:
```
val Right(files) = result
```
It will fail with a match error if it's a `Left`. Don't bother changing the PR for this alone, just a small note.